definitions/v3/blutopia-api.yml

```diff
@@ -54,10 +54,19 @@ search:
   paths:
     # https://hdinnovations.github.io/UNIT3D-Community-Edition-Docs/api_endpoints.html
     # https://github.com/HDInnovations/UNIT3D-Community-Edition/blob/master/app/Http/Controllers/API/TorrentController.php
-    - path: "/api/torrents/filter?api_token={{ .Config.apikey }}&name={{ if .Query.IMDBID }}{{ else }}{{ .Keywords }}{{ end }}{{ if .Query.TMDBID }}&tmdbId={{ .Query.TMDBID }}{{ else }}{{ end }}{{ if .Query.IMDBIDShort }}&imdbId={{ .Query.IMDBIDShort }}{{ else }}{{ end }}{{ if  .Query.TVDBID }}&tvdbId={{ .Query.TVDBID }}{{ else }}{{ end }}&sortField={{ .Config.sort }}&sortDirection={{ .Config.type }}&perPage=100&page=1{{ range .Categories }}&categories[]={{.}}{{end}}{{ if .Config.freeleech }}&free=1{{ else }}{{ end }}"
+    - path: "/api/torrents/filter?sortField={{ .Config.sort }}&sortDirection={{ .Config.type }}&perPage=100&page=1{{ range .Categories }}&categories[]={{.}}{{end}}{{ if .Config.freeleech }}&free=1{{ else }}{{ end }}"
       response:
         type: json
         attribute: attributes
+  inputs:
+  # if we have an id based search, add Season and Episode as query in name for UNIT3D < v6.  Else pass S/E Params for UNIT3D >= v6
+    api_token: "{{ .Config.apikey }}"
+    name: "{{ if or (.Query.TVDBID) (.Query.IMDBID) (.Query.TMDBID) }}{{ if .Query.Season }}S{{ .Query.Season }}{{else}}{{ end }}{{ if .Query.Ep }}E{{ .Query.Ep }}{{else}}{{ end }}{{else}}{{ .Keywords }}{{ end }}"
```

Comment:
We'd need to double-check this actually works in Jackett, see https://github.com/Jackett/Jackett/issues/3388

Reply:
It doesn't work for Prowlarr it appears, so I assume it won't for Jackett either.

```
2021-12-21 13:07:42.9|Info|Cardigann|Adding request: https://blutopia.xyz/api/torrents/filter?sortField=created_at&sortDirection=desc&perPage=100&page=1?api_token=(removed)&name=%7B%7Belse%7D%7D%7B%7B+end+%7D%7D&tmdbid=&tvdbid=&imdbid=
2021-12-21 13:07:42.9|Debug|Cardigann|Downloading Feed https://blutopia.xyz/api/torrents/filter?sortField=created_at&sortDirection=desc&perPage=100&page=1?api_token=(removed)&name=%7B%7Belse%7D%7D%7B%7B+end+%7D%7D&tmdbid=&tvdbid=&imdbid=
2021-12-21 13:07:43.1|Warn|Cardigann|Unable to connect to indexer

[v0.1.8.1265] System.NullReferenceException: Object reference not set to an instance of an object.
   at NzbDrone.Core.Indexers.Cardigann.CardigannRequestGenerator.DoLogin() in D:\a\1\s\src\NzbDrone.Core\Indexers\Definitions\Cardigann\CardigannRequestGenerator.cs:line 170
   at NzbDrone.Core.Indexers.Cardigann.Cardigann.DoLogin() in D:\a\1\s\src\NzbDrone.Core\Indexers\Definitions\Cardigann\Cardigann.cs:line 159
   at NzbDrone.Core.Indexers.HttpIndexerBase`1.FetchIndexerResponse(IndexerRequest request) in D:\a\1\s\src\NzbDrone.Core\Indexers\HttpIndexerBase.cs:line 388
   at NzbDrone.Core.Indexers.HttpIndexerBase`1.FetchPage(IndexerRequest request, IParseIndexerResponse parser) in D:\a\1\s\src\NzbDrone.Core\Indexers\HttpIndexerBase.cs:line 297
   at NzbDrone.Core.Indexers.HttpIndexerBase`1.TestConnection() in D:\a\1\s\src\NzbDrone.Core\Indexers\HttpIndexerBase.cs:line 456
```

Reply:
Use `Query.Episode` and get rid of the nesting. That variable gives the built out Episode string

Reply:
Yep, `Query.Episode` works well.

Reply:
done